frontend/src/context/analysisResultStateSlice.ts

```diff
@@ -317,6 +443,40 @@ const createAPIRequestParams = (
   return apiRequest;
 };
 
+function mergeTableRows(tableRows: TableRow[]) {
+  const mergedObject: Record<string, any> = {};
+
+  tableRows.forEach(obj => {
+    Object.keys(obj).forEach(objectKey => {
+      if (typeof obj[objectKey] === 'number') {
+        // eslint-disable-next-line fp/no-mutation
+        mergedObject[objectKey] = mergedObject[objectKey]
+          ? mergedObject[objectKey] + obj[objectKey]
+          : obj[objectKey];
+      } else {
+        // eslint-disable-next-line fp/no-mutation
+        mergedObject[objectKey] = obj[objectKey];
+      }
+    });
+  });
+
+  // TEMPORARY LOGIC TO DEDUP POPULATION COUNTS FOR WIND BUFFERS.
```

Comment:
@wadhwamatic should we move this logic to the backend instead of the frontend? What is your preferred behavior?

Reply:
Same with the rounding of values, should this happen in the backend?

Reply:
keeping it here for now as suggested by @wadhwamatic